### DISCUSSION TITLE
Word embeddings for ogbl-collab

### DISCUSSION THREAD

**Discussion by ChatterjeeAyan** [snap-stanford/ogb#380] [General]
In ogbl-collab, all nodes come with 128-dimensional features, obtained by averaging the word embeddings of papers that are published by the authors. Got a few questions regarding this:
1. What kind of word embedding is used here? Word2vec?
2. Do you use a pre-trained Word2ve for the purpose? Is that model open-source? 
3. Is the corpus of the papers available on which embeddings were created?

**Comment by weihua916** [comment]
Hi!

1. It is word2vec.
2. The model is trained on academic paper. The model is not open-sourced. It was trained a long time ago, and I do not think we can open-source the model.
3. No, it is not available.
